lib/tapioca/dsl/pipeline.rb

```diff
@@ -152,6 +152,9 @@ def rbi_for_constant(constant)
 
           compiler = compiler_class.new(self, file.root, constant)
           compiler.decorate
+        rescue
+          $stderr.puts("ERROR: `#{compiler_class.name}` failed to generate RBI for `#{constant}`")
```

Comment:
In other places we've been using `Error`:

```suggestion
          $stderr.puts("Error: `#{compiler_class.name}` failed to generate RBI for `#{constant}`")
```